pyproject.toml

```
@@ -18,7 +18,8 @@ dependencies = [
   "fastapi-users[beanie, oauth] == 10.4.0",
   "fastapi-versioning == 0.10.0",
   "MarkupSafe == 2.0.1",
-  "motor == 3.3.2",
+  "pymongo == 4.9.0",
+  "motor == 3.6.0",
```

Does it work if we keep packages in alphabetical order or do we need to pin `pymongo` first?
```suggestion
  "motor == 3.6.0",
  "pymongo == 4.9.0",
```

---

Yes, seems there is no requirements, so we can reorder it alphabetically